inventories/group_vars/all

```diff
@@ -344,6 +344,8 @@ grouper:
         - server_name: "{{ nginx_ssl.server_name }}"
           return: "https://$host$request_uri"
   loader:
+    compose_service: grouper_loader
+    container_name: grouper-loader
```

Comment:
Hmm, possibly. I'm a little bit ambivalent about it though, because I like the idea of having all of the group variables documented in one place.

I'm actually a little tempted to make the `deploy-grouper` playbook public instead. Since Grouper is required by the DE, it would be nice to provide a way to obtain Grouper with a "standard" configuration, which would be something like the installation that we have. There's no way we could ever cover all possible Grouper configurations, but creating a reference configuration might be handy.